modules/playing-cards/assets/js/frontend/handlers/playing-cards-handler.js

```diff
@@ -0,0 +1,43 @@
+export default class PlayingCardHandler extends elementorModules.frontend.handlers.Base {
+	onInit() {
+		super.onInit();
+		this.elements = this.getDefaultElements();
+		this.clickHandler = this.onClick.bind( this );
+	}
+	getDefaultSettings() {
+		const cardItemPrefix = 'e-playing-cards-item';
+		return {
+			selectors: {
+				playingCardContainer: '.e-playing-cards',
+				playingCardItem: `.${ cardItemPrefix }`,
+			},
+			classes: {
+				playingCardBack: `${ cardItemPrefix }-back_suit`,
+			},
+		};
+	}
+
+	getDefaultElements() {
+		const { selectors } = this.getSettings();
+
+		return { $playingCardContainer: this.findElement( selectors.playingCardContainer ) };
+	}
+
+	bindEvents() {
+		this.elements.$playingCardContainer.on( 'click', this.clickHandler );
+	}
```

Comment:
What is the difference between this bindEvent and the bind onInit?

Reply:
I assign a bound method to a property of the object so that I can use it later in bindEvents when adding the handler to a DOM element. As I understand, bindEvents and unbindEvents are like "hooks" where adding handlers should ideally be placed.

Why did I store the bound method in onInit? Because I haven’t dealt with webpack/babel configs (not sure if they use the transform-class-properties plugin) and I’m unsure if I can write methods as class properties (method = () => {}).

Also, this approach makes it more convenient to verify in tests that the correct handler was added to the DOM element. Since this.method.bind(this) returns a new function, it’s impossible to check in tests that the onClick method of the object was added as a handler for the DOM element. And I didn’t want to write expect.any(Function).